changelogs/sanity_fix_future_boilerplate.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
My understanding of antsibull-changelog is that we want to keep fragments under changelogs/fragments, could you move this into that directory please?  Otherwise this lgtm, thanks!